packages/frontend/src/stores/workflows.ts

```diff
@@ -0,0 +1,70 @@
+import { useWorkflow } from "@/composables/useWorkflow";
+import { useSDK } from "@/plugins/sdk";
+import { defineStore } from "pinia";
+import type { Workflow } from "shared";
+import { computed, ref } from "vue";
+
+export interface WorkflowState {
+  workflows: Workflow[];
+  installedWorkflowsNames: string[];
+  searchQuery: string;
+}
+
+export const useWorkflowStore = defineStore("workflows", () => {
+  const sdk = useSDK();
+  const { installedWorkflows } = useWorkflow();
+
+  // State
+  const state = ref<WorkflowState>({
+    workflows: [],
+    installedWorkflowsNames: [],
+    searchQuery: "",
+  });
```

Comment:
This should be either three refs:
```
{
  workflows: ref([])
  installedWorkflowsNames: ref([])
  searchQuery: ref("")
}
```

or a reactive object
```
const state = reactive({
  workflows: [],
  installedWorkflowsNames: [],
  searchQuery: ""
});
```

If you keep it as-is, there's no guarantee that modifying nested fields like `state.value.workflows.push(...)` will be reactive.

On the other hand:
```
state.workflows.value.push(...)
```

... will be reactive